<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLineEdit" name="leUsername">
                <property name="placeholderText">
                 <string>Login</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lePassword">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string extracomment="Password">Password</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="btnRegistration">
                <property name="text">
                 <string>Registration</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnAuth">
                <property name="text">
                 <string>Authorization</string>
                </property>
                <property name="autoDefault">
                 <bool>true</bool>
                </property>
                <property name="default">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QListView" name="lvTitleList">
              <property name="dragEnabled">
               <bool>false</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::NoDragDrop</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::CopyAction</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="btnAddPost">
                <property name="toolTip">
                 <string>Add New Post</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDeletePost">
                <property name="toolTip">
                 <string>Delete Post</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLineEdit" name="leTitle"/>
            </item>
            <item>
             <widget class="QTextEdit" name="textEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionDownload_Database"/>
    <addaction name="actionUpload_Database"/>
    <addaction name="separator"/>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuFIle"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="separator"/>
   <addaction name="actionTextLeft"/>
   <addaction name="actionTextCenter"/>
   <addaction name="actionTextRight"/>
   <addaction name="actionTextJustify"/>
  </widget>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/Images/format_bold.png</normaloff>:/Images/format_bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/Images/format_italic.png</normaloff>:/Images/format_italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/Images/format_underlined.png</normaloff>:/Images/format_underlined.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionTextLeft">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/Images/format_align_left.png</normaloff>:/Images/format_align_left.png</iconset>
   </property>
   <property name="text">
    <string>TextLeft</string>
   </property>
  </action>
  <action name="actionTextCenter">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/Images/format_align_center.png</normaloff>:/Images/format_align_center.png</iconset>
   </property>
   <property name="text">
    <string>TextCenter</string>
   </property>
  </action>
  <action name="actionTextRight">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/Images/format_align_right.png</normaloff>:/Images/format_align_right.png</iconset>
   </property>
   <property name="text">
    <string>TextRight</string>
   </property>
  </action>
  <action name="actionTextJustify">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/Images/format_align_justify.png</normaloff>:/Images/format_align_justify.png</iconset>
   </property>
   <property name="text">
    <string>TextJustify</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/Images/exit_to_app.png</normaloff>:/Images/exit_to_app.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionDownload_Database">
   <property name="text">
    <string>&amp;Download Database</string>
   </property>
  </action>
  <action name="actionUpload_Database">
   <property name="text">
    <string>&amp;Upload Database</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>&amp;Logout</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
